.github/workflows/docker.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Verilator Container
+
+on:
+  push:
+    tags: [ 'v*' ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        contexts:
+          - "ci/docker/run:verilator"
+          # - "ci/docker/buildenv:verilator-buildenv"
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Extract context variables
+      run: |
+        echo "${{ matrix.contexts }}" | sed -r 's/(.*):.*/build_context=\1/' >> "$GITHUB_ENV"
+        echo "${{ matrix.contexts }}" | sed -r 's/.*:(.*)/image_name=\1/' >> "$GITHUB_ENV"
+        echo "git_tag=${GITHUB_REF#refs/*/}" >> "$GITHUB_ENV"
+
+    - name: Docker meta
+      id: docker_meta
+      uses: docker/metadata-action@v4
+      with:
+        images: |
+          ${{ secrets.DOCKER_HUB_NAMESPACE }}/${{ env.image_name }}
+        tags: |
+          type=match,pattern=(v.*),group=1
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```

Comment:
Does docker require QEMU?  Seems surprising.

Reply:
This is required for doing multi-platform builds (amd64, arm64). Checkout [this page](https://docs.docker.com/build/ci/github-actions/multi-platform/).

Reply:
Are we building multiplatform?

Reply:
Yes, I configured it to build for `linux/amd64` and `linux/arm64`
https://github.com/verilator/verilator/pull/4163/files#diff-3f5366f6d6df3ec1179e5efadc6f350bfa88eebf4e2da589b4d94ccb85ae5e94R59